resources/blockchain_network_config.json

```diff
@@ -9,7 +9,7 @@
     "network_id":"42"
   },
   "ropsten":{
-    "ethereum_json_rpc_endpoint":"https://ropsten.infura.io",
+    "ethereum_json_rpc_endpoint":"https://ropsten.infura.io/v3/09027f4a13e841d48dbfefc67e7685d5",
```

Comment:
Why not common SingularityNet project-id?

Reply:
kept this for testing alone ..